sentinel-core/src/main/java/com/alibaba/csp/sentinel/Tracer.java

```diff
@@ -110,7 +110,7 @@ public static void traceEntry(Throwable e, int count, Entry entry) {
             return;
         }
 
-        DefaultNode curNode = (DefaultNode)entry.getCurNode();
+        DefaultNode curNode = (DefaultNode) entry.getCurNode();
```

Comment:
Maybe the reformat for unrelated other modules is necessary?

Reply:
emm... i just do it when i see it.

Reply:
This PR is for adapter modules, so it's better not to contain other unnecessary changes (so that the PR is clear enough). Maybe we need another PR to do this :)

Reply:
I get it. Do i need to  restore unconcerned file now?  I just modify a small number of files.